### PR TITLE
Fix erf, atan2 for integers

### PR DESCRIPTION
**Pull request by qihqi**
Fixes:
https://github.com/pytorch/xla/issues/5863
https://github.com/pytorch/xla/issues/5852

### CODE REVIEW

**Review by wonjoolee95 — COMMENTED** [review]
Changes LGTM, but seems like the `test_aten_atan2_1` unit test is failing. I assume you didn't want to enable this test yet?

**Comment by qihqi** [comment]
> Changes LGTM, but seems like the `test_aten_atan2_1` unit test is failing. I assume you didn't want to enable this test yet?

fixed tests

File: test/test_core_aten_ops.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ def diff_output(testcase, output1, output2, rtol, atol, equal_nan=True):
     output2_cpu = output2.detach().cpu()
     if output2_cpu.dtype != output1.dtype:
       output2_cpu = output2_cpu.to(output1.dtype)
+    # import pdb; pdb.set_trace()
```
Comment (wonjoolee95) [inline test/test_core_aten_ops.py:32]: nit: remove debugging code

**Review by wonjoolee95 — APPROVED** [review]
LGTM, thanks!

File: test/test_core_aten_ops.py (hunk 1 above)
Comment (qihqi) [reply test/test_core_aten_ops.py:32]: Let's leave it there... I always put it back for debugging for every single op worked. So its easier to uncomment for the next time. Lets remove after all ops are done XD

**Pull request merged**